index.js

```diff
@@ -0,0 +1,24 @@
+const menuBtn = document.querySelector('.menu-btn');
+const headerNavMobile = document.querySelector('.header-nav-mobile');
+const cancelCross = document.querySelector('.cancel-cross');
+const linkPortfolio = document.querySelectorAll('.menu-link');
+//const linkAbout = document.querySelector('.menu-link');
+//const linkContact = document.querySelector('.menu-link');
```

Comment:
**OPTIONAL** - Please remove all commented code from your js file, this will help to keep your code neat and easy to maintain 👍 👍